buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec

```diff
@@ -111,6 +112,7 @@ Requires: python3-sip-devel
 Requires: python3-PyQt4-devel
 Requires: python-qt5-devel
 Requires: python3-QtPy
+Requires: python3-qtawesome
```

Comment:
Correct the name and only use on fedora with ` %{?fedora:Requires: python3-QtAwesome}`

Reply:
Thanks. Just for my own learning why don't the python3 packages like `QtPy` need this restriction too?